ps.go

```diff
@@ -73,9 +85,9 @@ func printContainers(ctx context.Context, clicontext *cli.Context, containers []
 		if _, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\t%s\n",
 			id,
 			imageName,
-			"", // COMMAND
-			"", // CREATED
-			"", // STATUS
+			containerCommand(spec, true),
```

Comment:
This should take clicontext.Bool("no-trunc") as the second arg?

Reply:
updated